src/corelibs/U2View/src/ov_sequence/find_pattern/FindPatternWidget.cpp

```diff
@@ -678,15 +678,19 @@ void FindPatternWidget::updateErrorLabelState() {
                               messageFlagMap.contains(SequenceIsTooBig) ||
                               (messageFlagMap.size() == 1 && messageFlagMap.contains(PleaseInputAtLeastOneSearchPatternTip));
     if (hasNoPatternErrors) {
-        GUIUtils::setWidgetWarning(textPattern, false);
+        GUIUtils::setWidgetWarningStyle(textPattern, false);
     }
 }
 
 QString FindPatternWidget::buildErrorLabelHtml() const {
+    QString errorColor = Theme::errorColorLabelHtmlStr();
```

Comment:
`static const QString ERROR_COLOR` and so on below

Reply:
This is wrong. Theme is a dynamic object and should be re-read on use.